ci-operator/step-registry/ipi/conf/azurestack/ipi-conf-azurestack-commands.sh

```diff
@@ -45,5 +43,7 @@ compute:
   replicas: ${workers}
 EOF
 
+sed -i "s/${cluster_name}/${NAMESPACE}/g" ${CONFIG}
```

Comment:
why are we doing this (and why can't we just write the correct value in the heredoc)?

Reply:
I agree this isn't ideal. Can't think of any other way to replace the existing metadata value to this without extra libraries.

Reply:
Why can't we keep the cluster name as `${NAMESPACE}-${JOB_NAME_HASH}`? I can't remember if we discussed this in the previous PR.

Reply:
Ah forgot to mention that. I think Azurestack has a 24 character restriction for their resource names.

Reply:
I was able to create a resource group with a name I stole from Azure CI: `ci-op-0k479mr9-9ee9d-z628v-rg` so that character limit may only be for certain resources. If you have a moment, it would be good to narrow down where having the complete name would fail or which resource has the limit. 

Then add a comment to the code explaining why we need to do this `sed` here. 

That said, ASH CI is very small so I'm fine with the current solution. If you don't have time to get to this soon, I'll slap a lgtm on here.

Reply:
Changed it and it's not working
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/23453/rehearse-23453-pull-ci-openshift-installer-release-4.10-e2e-azurestack-upi/1460622467683848192

Reply:
I believe the restriction is on the storage account name. 

cf how we are handling the storage account name in public azure:
https://github.com/openshift/release/pull/22320/files#diff-5483a633ffe74be033f28def4e9b0c9adbfb0ec96b53e43ef314e22672b64ae9R120-R124